lib/create-environment-api.ts

```diff
@@ -1152,11 +1152,13 @@ export default function createEnvironmentApi(makeRequest: MakeRequest) {
      */
     createUpload: function createUpload(data: { file: string | ArrayBuffer | Stream }) {
       const raw = this.toPlainObject() as EnvironmentProps
+      const envId = raw.sys.aliasedEnvironment?.sys.id || ''
```

Comment:
I think this is not targeting the public environment id. As far as I understand the code this either:

1. in the case that we are looking at an alias route, using the underlying environment id

2. in the case that no aliases are used, just providing an empty string.

Not sure about all the details of how the upload endpoint has been adjusted, but I am assuming it will always use either the alias-id or the environment-id, but never the environment-id that is being targeted by the alias, not empty string.